ndlarflow/h5_to_root_ndlarflow.py

```diff
@@ -76,6 +76,113 @@ def main(argv=None):
     if overrideOutname==1:
         outname = fileNames[0].split('/')[-1]+'_hits_uproot.root'
 
+    ## We are choosing to write a bogus subevent to set the types of all the branches.
+    ## The hope is this will then work well even in the case where the first event we'd see is actually a bad event
+    ##################################################
+    if len(fileNames) > 0:
+        # Simple versions of the input vectors where everything is set to 0 of the proper type
+        hits_z = np.array([0.]).astype('float32')
+        hits_y = np.array([0.]).astype('float32')
+        hits_x = np.array([0.]).astype('float32')
+        hits_Q = np.array([0.]).astype('float32')
+        hits_E = np.array([0.]).astype('float32')
+        hits_ts = np.array([0.]).astype('float32')
+        runID = np.array( [0], dtype='int32' )
+        subrunID = np.array( [0], dtype='int32' )
+        eventID = np.array( [0], dtype='int32' )
+        triggerID = np.array( [0], dtype='int32')
+        event_start_t = np.array( [-5], dtype='int32' )
```

Comment:
Just a reminder of what we discussed earlier: the initialization could be made uniform across arrays